src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -98,10 +98,10 @@ public void resetData(ReadOnlyAddressBook newData) {
         } catch (DuplicateInsuranceException e) {
             assert false : "AddressBooks should not have duplicate insurances";
         }
-        syncMasterLifeInsuranceMapWith(persons);
+        syncMasterLifeInsuranceMap();
```

Comment:
What is the reason of changing the name of this function? I followed the way `syncMasterTagListWith(persons)` is defined and also believe that the original name is more meaningful in the sense that it tells you the target I want to sync with.

Reply:
but this function is only called within the addressbook and has no meaning outside of it. Since this is only to link the person n insurances WITHIN the addressbook, there should be no target as the only target is itself. Even if you want to load from another addressbook, you will replace all the data within the current addressbook and re-sync them WITHIN the addressbook again so there is no need for a target

Reply:
Also i decided to change because i see no sense in a class function passing its own class attribute to another method within the class itself

Reply:
If we stick with your reasoning, then the `syncMasterTagListWith(Person person)` and `syncMasterTagListWith(UniquePersonList persons)` should also be changed. I did some research online and [find it controversial](https://softwareengineering.stackexchange.com/questions/146508/passing-member-variable-as-a-method-parameter). I propose we stick with the current version and ask the TA on Thursday. What do you think?

Reply:
haha okies =D, i noticed the same with tag but i didnt change haha